README.md

```diff
@@ -1 +1,62 @@
-# WEB-SUBMIT-2022-ACCRA
\ No newline at end of file
+# Web Summit 2022 Accra
+
+Live preview [here](https://jerryowusu.github.io/Web-Summit-Accra/). 
+
+## Screenshots
+![Example screenshot](/src/images/desktop-screenshot.png)
```

Comment:
- [ ] The `screenshot` in the README file is broken. I would highly recommend fixing it to make README professional.

Reply:
Thanks for drawing my attention. It has been resolved.